plugins/arangodbendorsements/arangodb.go

```diff
@@ -119,8 +119,8 @@ func (e *ArangoStore) IsInitialised() bool {
 }
 
 // GetQueryParam returns a suitable query parameter based on supplied input
-func (e *ArangoStore) GetQueryParam(input interface{}) (string, error) {
-	switch input.(CollectionType) {
+func (e *ArangoStore) GetQueryParam(input string) (string, error) {
```

Comment:
maybe instead of `input`, which has no particular meaning, use `queryParamID` or similar?

Reply:
ID looks a little odd as it is a string, may be simple to name it as qParam ?

Reply:
the question is: what is this `input` parameter?  is it a query parameter (qParam)? or is it a unique identifier for the query parameter (qParamID)?

Reply:
Resolved it by renaming input to qParamID